app/services/exchange_rates/update_currency_rates_service.rb

```diff
@@ -1,8 +1,9 @@
 module ExchangeRates
   class UpdateCurrencyRatesService
-    def initialize(date, sample_date)
+    def initialize(date, sample_date, type)
       @date = date
```

Comment:
can we add reader methods for these?

Reply:
I just followed the existing code style of this class as nothing currently in this class is using attr_reader